is-array/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for is-array 1.0
+// Project: https://github.com/retrofox/is-array
+// Definitions by: Pine Mizune <https://github.com/pine>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = isArray;
+declare function isArray(val?: any): boolean;
```

Comment:
Sure, the return type of `Array.isArray` is the following:

```ts
isArray(arg: any): arg is Array<any>;
```

So it's the same thing. `arg is Array<any>` is a type predicate assertion, which basically says that

1. The function returns a `boolean`
2. If the function returns `true`, then whatever was passed in for `arg` is compatible with `Array<any>`.